client/state/sites/selectors.js

```diff
@@ -45,7 +45,7 @@ import { isSiteUpgradeable, getSiteOptions } from 'state/selectors';
  * @return {?Object}        Site object
  */
 export const getRawSite = ( state, siteId ) => {
-	return state.sites.items[ siteId ] || null;
+	return getSitesItems( state )[ siteId ] || null;
```

Comment:
nit: `get( getSitesItems( state ), siteId )` works the same way -- `null` is the default return value from `get` if nothing is found.

Reply:
lodash docs specific that `undefined` is returned when the property doesn't exist.